2020/scoring/clean_and_join_csvs.sh

```diff
@@ -82,14 +82,14 @@ if [[ ${PROCESS_INC} == "true" ]]; then
 fi
 
 if [[ ${PROCESS_MV} == "true" ]]; then
-    TMP_MV_2019_ORDERED=$(mktemp).mv_2019_ordered
+    TMP_MV_2019_ORDERED=$(mktemp --tmpdir mv_2019_ordered.XXXXXX)
```

Comment:
Annoying... macOS doesn't support --tmpdir.  Is it absolutely necessary to do this change?